_maps/map_files/RandomRuins/SpaceRuins/syndie_space_base.dmm

```diff
@@ -1577,15 +1577,13 @@
 /turf/simulated/floor/plating,
 /area/ruin/unpowered/syndicate_space_base/engineering)
 "rp" = (
-/obj/machinery/economy/vending/toyliberationstation{
-	req_access_txt = "150"
-	},
 /obj/structure/cable{
 	d1 = 1;
 	d2 = 2;
 	icon_state = "1-2";
 	tag = ""
 	},
+/obj/machinery/economy/vending/toyliberationstation/secured,
```

Comment:
I dont think this access check worked anyways since anyone could get stuff from the vendor (i think), but you may wish to readd this access requirement

Reply:
Can confirm, the access check never worked.